src/map.cpp

```diff
@@ -775,12 +775,10 @@ void MMVManip::initialEmerge(v3s16 p_min, v3s16 p_max, bool load_if_inexistent)
 			(p_min*MAP_BLOCKSIZE, (p_max+1)*MAP_BLOCKSIZE-v3s16(1,1,1));
 
 	u32 size_MB = block_area_nodes.getVolume() * sizeof(MapNode) / 1000000U;
-	if(size_MB >= 1)
-	{
-		infostream<<"initialEmerge: area: ";
+	if (size_MB >= 4) {
```

Comment:
Unrelated change?

Reply:
The commit is literally named "unrelated refactor", so: yes.